docs/source/tutorial/package_a_project.md

```diff
@@ -35,57 +35,45 @@ or you separate "source" and "build" directories within the root path.
 > Separate source and build directories (y/n)? [n]:
 ```
 
-In the following instructions, we assume that you select `y` to separate the built files from the source files, but you can choose either option safely and continue to follow the process.
-
-Enter any additional information that Sphinx requests such as the project name and the documentation language (which defaults to English).
-
+Select `y` to separate the built files from the source files, and enter any additional information that Sphinx requests such as the project name and the documentation language (which defaults to English).
```

Comment:
Encouraging the users to separate `build` from `source`, which is backwards compatible and also a good practice. I think it's not too bad, expert users will do whatever they want anyway.